All EIPIP Meetings/Meeting 007.md

```diff
@@ -0,0 +1,527 @@
+
+So there is basically a fine line between the hard fork process and the EIP process itself. I tried to document that sometime back in one of the EIPs - it must have been moved somewhere else. But yeah, I find that this may be a good way forward in which people will be getting accurate information even about all the clients and where they stand at any point in time.
+
+**Hudson Jameson**
+Okay, so we've kind of agreed that the process of hard fork management and EIP development should be separate, or implementation should be separate, right?
+
+**James Hancock**
+The EIP standardization and the network upgrade deployment should be separate.
+
+**Hudson Jameson**
+OK, so we're all there. We're saying that there might need to be new tools in order to make that happen. We're also saying there might need to be, or that there is value in having some type of asynchronous coordination beyond the AllCoreDevs Gitter chat for people who are in different time zones, and that there might be tools we need to bring up to do that. I think I got that all right. Is there anything I got wrong?
+
+**Pooja Ranjan**
+We already have existing tools. We just need to enforce the acceptance and implementation of that. We have a discussion port on the implementation. In my understanding, most of the EIPs are there, and they are referred to be as a point of discussion. But sometimes what happens is the discussion shifts to the pull request. And there we lost the track. So if we can make our standard process a standard place, that this is it, and this is where you need to go, and we refer the link in AllCoreDev meetings that might bring all the information to one place.
+
+ **Hudson Jameson**
+And that would be a change to EIP 1. Because in EIP 1, we say you should probably use EthMagicians. But you can use Reddit and Issues and other stuff.
+
+**Pooja Ranjan**
+Yeah, it's a kind of a recommendation, but I think to make it standardized, enforcement would be required.
+
+ **Hudson Jameson**
+Yeah, and I don't think that's a bad thing.
+
+**jpitts**
+Yeah, I agree with that. This is Jamie - I dropped in part way through the meeting, but, yeah, I think that's actually a really good idea. I've been thinking about that myself is to have a place that links to all kinds of different related discussions, but in a single URL. And it could be per EIP or per a meta EIP or hard fork EIP. I'm not sure where you would put such a URL or such a resource, but it's important, because there's even Gitter chat links. And you know, there's just so many places where decisions are being made or discussions are happening.
+
+ **Hudson Jameson**
+Yes.
+
+**Pooja Ranjan**
+It can be for EIPs in the beginning, because the meta EIP for any upgrade would be the next step. So we can start with going individual EIP-wise, and then we can also create a thread inventory - just a link over here on discussion should be at one place with the EIPs only. That's my thought process.
```

Comment:
It can be for EIPs in the beginning, because the meta EIP for an upgrade would be the next step. So we can start with going individual EIP-wise, and then we can also create a thread for Meta EIP - just a link, the discussion should be in one place with the EIPs only. That's my thought process.